packages/admin-portal/src/components/apps/toggle-featured.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FC, useEffect } from 'react'
+import { elMt5, FormLayout, InputWrap, Label, ToggleRadio } from '@reapit/elements'
+import { useForm, UseFormWatch } from 'react-hook-form'
+import { SendFunction, useReapitUpdate } from '@reapit/utils-react'
+import { UpdateActionNames, updateActions } from '@reapit/utils-common'
+import { reapitConnectBrowserSession } from '../../core/connect-session'
+import { AppSummaryModelPagedResult } from '@reapit/foundations-ts-definitions'
+
+export interface ToggleFeaturedForm {
+  isFeatured?: 'FEATURED' | 'NOT_FEATURED'
```

Comment:
Just a small comment, I'd expect `isSomething` to be a boolean? Probably too late to change now but `featureType` might be better as a property name? But obviously this is a bit pedantic

Reply:
isFeatured is the filter on the API. It's a boolean. But I am using a radio button and all input values are strings... I could set it to be 'true' or 'false' but that's equally gross